modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -1069,11 +1074,23 @@ func (h *HTTPTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 			}
 
 		case "forward_proxy_url":
+			caddy.Log().Warn("The 'forward_proxy_url' field is deprecated. Use the 'network_proxy' field instead.")
 			if !d.NextArg() {
 				return d.ArgErr()
 			}
-			h.ForwardProxyURL = d.Val()
 
+			h.ForwardProxyURL = d.Val()
+		case "network_proxy":
```

Comment:
Newline between cases for consistency (and visual separation)

Also pls remove the newline before this line, Github isn't letting me put in a suggestion for that :sob:

```suggestion
			h.ForwardProxyURL = d.Val()

		case "network_proxy":
```

As an aside, maybe just make `forward_proxy_url` set up a `NetworkProxyRaw` with `url` module right away? We can make Caddyfile adapter do the "right thing" in the meantime and handling of this option & for JSON later.

Reply:
I can do that.

Reply:
Done for both caddyfile and JSON